OpenRA.Mods.Common/Scripting/Global/WorldActorConditionGlobal.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq;
+using Eluant;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Scripting;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptGlobal("WorldActor")]
+	public class WorldActorConditionGlobal : ScriptGlobal
```

Comment:
This is mostly redundant against `ConditionProperties`. You just need to add a `WorldActor` property to `MapGlobal`.

Reply:
`WorldActorConditionGlobal` is just part of testcase, it is unrelated to the actual changing. 


We can add that property in another PR, though.

Reply:
I filled #21188 on `ConditionProperties` on WorldActor